### ISSUE TITLE
Unicode (non-ascii) characters not passed to password box in UWP

### ISSUE THREAD

**Issue by RafalSkorka** [Microsoft/WinAppDriver#151]
I have two XAML controls in UWP project: PasswordBox and TextBox and I do a 
```
string password = "zażółćgęśląjaźń";
driver.FindElementByAccessibilityId("passwordBox").SendKeys(password);
driver.FindElementByAccessibilityId("textBox").SendKeys(password);

```
the result is PasswordBox displays "zaglja" and TextBox displays "zażółćgęśląjaźń" (ie. properly). See below:
![passwordtextbox](https://cloud.githubusercontent.com/assets/3476094/23067560/311e1ce6-f520-11e6-9486-681f567e0417.png)
Is this a bug or I should send the string in a different way to PasswordBox?

**Comment by RafalSkorka** [comment]
I found a workaround which does not send keys through the driver:

```
System.Windows.Forms.Clipboard.SetText(password);
driver.Keyboard.SendKeys(OpenQA.Selenium.Keys.LeftControl + "v" + OpenQA.Selenium.Keys.LeftControl);

```
but anyway this should probably be fixed.

**Comment by timotiusmargo** [comment]
This issue is already fixed and fully addressed in [v1.0](https://github.com/Microsoft/WinAppDriver/releases/tag/v1.0) release. Thanks for bringing it up 👍
